app/src/main/java/org/projectbuendia/client/ui/BaseActivity.java

```diff
@@ -110,6 +111,22 @@ private void initializeWrapperView() {
             (FrameLayout) mWrapperView.findViewById(R.id.status_wrapper_inner_content);
         mStatusContent =
             (FrameLayout) mWrapperView.findViewById(R.id.status_wrapper_status_content);
+
+        if (snackBar == null) {
+            snackBar = new SnackBar(mWrapperView);
+        }
+    }
+
+    public int snackBar(String message) {
+        return snackBar.message(message);
+    }
+
+    public int snackBar(String message, String actionMessage, View.OnClickListener listener) {
+        return snackBar.message(message, actionMessage, listener, 999);
+    }
+
+    public int snackBar(String message, String actionMessage, View.OnClickListener listener, int priority) {
```

Comment:
@capnfabs I'm going to put this on Java Docs. 0 is the highest and 999 the default (low). What do you think?
This is to make things easy to implement. (ordering).